### PR TITLE
HAB shell and Sensor command

### PR DESCRIPTION
**Pull request by krisbellemans**
This pull request:

- Implements a basic line oriented command interpreter which allows for adding/removing commands at runtime
- Implements the HAB shell
- Implements the pressure sensor command and adds it to the HAB shell

### CODE REVIEW

**Comment by krisbellemans** [comment]
@gitlynx, ik heb hier alvast een PR aangemaakt, maar is nog heel WIP.

File: firmware/NOKHAB/hab-server/hab_server.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import sys
+import threading
+
+from hab_logger import LoggerBase
+from hab_shell import HabShell
+from sensor import Sensor, SensorCmd
+from threading import Thread
+
+HOST = '0.0.0.0'
+PORT = 1234
```
Comment (gitlynx) [inline firmware/NOKHAB/hab-server/hab_server.py:15]: Why this? 
Why not using telnet daemon module? and related port?

Comment (gitlynx) [reply firmware/NOKHAB/hab-server/hab_server.py:15]: feedback: This implementation can work with telnet client

**Review by gitlynx — APPROVED** [review]
Pair review done. Accepted. Changes in future commits.

**Pull request merged**